e2e/utils/_.bash

```diff
@@ -21,6 +21,8 @@ install_shared_asset() {
 }
 
 standard_setup() {
+    E2E_CACHE_ROOT="$(dfx cache show)"
+    export E2E_CACHE_ROOT
```

Comment:
Please revert this, since we've handled it elsewhere

Reply:
Good catch - I was using this locally and committed it accidentally.  Done.